Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -104,31 +112,68 @@ import kotlinx.collections.immutable.PersistentList
 import kotlinx.collections.immutable.toPersistentList
 import kotlinx.coroutines.launch
 
+@OptIn(ExperimentalMaterial3AdaptiveApi::class)
 @Composable
 fun Home(
     navigateToPodcastDetails: (PodcastInfo) -> Unit,
+    navigateToPlayer: (EpisodeInfo) -> Unit,
     viewModel: HomeViewModel = viewModel()
 ) {
     val viewState by viewModel.state.collectAsStateWithLifecycle()
-    Surface(Modifier.fillMaxSize()) {
-        Home(
-            featuredPodcasts = viewState.featuredPodcasts,
-            isRefreshing = viewState.refreshing,
-            homeCategories = viewState.homeCategories,
-            selectedHomeCategory = viewState.selectedHomeCategory,
-            filterableCategoriesModel = viewState.filterableCategoriesModel,
-            podcastCategoryFilterResult = viewState.podcastCategoryFilterResult,
-            library = viewState.library,
-            onHomeCategorySelected = viewModel::onHomeCategorySelected,
-            onCategorySelected = viewModel::onCategorySelected,
-            onPodcastUnfollowed = viewModel::onPodcastUnfollowed,
-            navigateToPodcastDetails = navigateToPodcastDetails,
-            onTogglePodcastFollowed = viewModel::onTogglePodcastFollowed,
-            onLibraryPodcastSelected = viewModel::onLibraryPodcastSelected,
-            onQueueEpisode = viewModel::onQueueEpisode,
-            modifier = Modifier.fillMaxSize()
-        )
+    val navigator = rememberSupportingPaneScaffoldNavigator<String>(
```

Comment:
All of these changes should be in the other Home composable, rather than here.

Reply:
I see you opened #1305. Let's address this issue in your PR